common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -226,13 +226,28 @@ def test_cc_button_wo_english_transcript(self):
         self.navigate_to_video()
         self.video.show_captions()
 
-        # Verify that we see "好 各位同学" text in the captions
+        # Verify that we see "好 各位同学" text in the transcript
         unicode_text = "好 各位同学".decode('utf-8')
         self.assertIn(unicode_text, self.video.captions_text)
 
-    def test_cc_button_transcripts_and_sub_fields_empty(self):
+    def test_cc_button(self):
+        """
+        Scenario: CC button works correctly with transcript in YouTube move
```

Comment:
typo? ... I think it's supposed to say  "YouTube mode" instead of "YouTube move".